pandas/core/frame.py

```diff
@@ -5919,6 +5921,14 @@ def dropna(
 
         agg_obj = self
         if subset is not None:
+            if (
+                not np.iterable(subset)
```

Comment:
We have is_list_like for such checks, would this do the trick?

Reply:
I attended your comments, could you have another look?

Reply:
Yes, this is what I meant, thx